ubuntu_dockerfile

```diff
@@ -0,0 +1,174 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##########################################################################
+
+# INSTRUCTIONS:
+#
+# This Dockerfile allows for an easy installation of Oppia for Windows
+# users and a more reliable testing environment for running test scripts.
+# 
+# Current known limitations:
+#   - no support for run_e2e_tests.sh
+#   - run_frontend_tests.sh might not run correctly every time
+#
+##########################################################################
+###################### TO SET UP THE OPPIA REPOSITORY ####################
+##########################################################################
+#
+# STEP 1
+# Download Docker Desktop for Windows here:
+# https://hub.docker.com/editions/community/docker-ce-desktop-windows
+#
+# STEP 2
+# Start Docker by clicking on the Docker application (a Docker icon should
+# appear in your taskbar tray).
+#
+# STEP 3
+# Build the Docker image from the Oppia root folder (note the "." at the
+# end of the command):
+#
+#   docker build -t {some_name} -f ubuntu_dockerfile .
+#      where you should replace {some_name} with whatever you want to call
+#      your Docker image.
+#   ex: docker build -t oppia_image -f ubuntu_dockerfile .
+#
+# Expect up to a 2 minute delay until the first line of output is printed.
+# Runtime: 15-25 minutes
+# If this is not your first time running this command, add --no-cache flag
+# to rebuild from the beginning:
+#   docker build -t {some_name} -f ubuntu_dockerfile . --no-cache
+#   ex: docker build -t oppia_image -f ubuntu_dockerfile . --no-cache
+#
+# STEP 4
+# Follow the instructions on how to start the Oppia server below.
```

Comment:
Delete this line and "step 4", it's superfluous

Reply:
Done.